src/frontends/common/include/openvino/frontend/place.hpp

```diff
@@ -302,5 +302,52 @@ class FRONTEND_API Place {
     /// \param another Another place object
     virtual bool is_equal_data(const Ptr& another) const;
 };
+
+/**
+ * \brief
+ *
+ */
+class GenericPlace : public Place {
+    std::shared_ptr<void> m_shared_object;  //!< Frontend library used  by place implementation.
```

Comment:
Will it be better/workable if replace library reference with frontend manager instance reference?

Reply:
Will check. From work I've done it looks like keeping frontend manager is not required for place object, so for now will stick to keep only shared_object.